.upptimerc.yml

```diff
@@ -159,6 +190,10 @@ sites:
   - name: sensors.AFRICA - API
     url: https://api.sensors.africa/docs
     icon: https://sensors.africa/favicons/green/favicon.ico
+    expectedStatusCodes:
```

Comment:
I'd rather you add a new endpoint used by sensors to send in data. That's the one we need to guard against. I don't think we can much if `/docs` redirects or not.